astore/rpc/auth.proto

```diff
@@ -26,9 +26,20 @@ message TokenResponse {
   repeated string cahosts = 6; // List of hosts the CA should be trusted for
 }
 
+message HostCertificateRequest {
+  repeated string hosts = 1; // List of dns names you want this server to be authorized to be. Does not have to be valid TLDs.
```

Comment:
As per comment in the chat: I think this should sign a certificate provided directly by the client, with the private key never sent over the wire.

I would s/HostCertificate/SignCertificate/g

Reply:
see comment in chat.